conf/app.routes

```diff
@@ -17,10 +17,12 @@ GET        /timeout
 GET         /:taxYear/summary                                                       controllers.DeductionsSummaryController.show(taxYear: Int)
 
 GET         /:taxYear/contractor                                                    controllers.ContractorSummaryController.show(taxYear: Int, contractor: String)
+GET         /:taxYear/add-cis-deduction                                             controllers.ContractorSummaryController.addCisDeduction(taxYear: Int, contractor: String)
```

Comment:
This is a bit of a strange endpoint. It saves data but it is a get request. It would be nice if it can be improved.